common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -833,6 +834,48 @@ public static int hashCodeAscii(CharSequence bytes) {
         return null;
     }
 
+    private static class Mpsc {
+        private static boolean USE_MPSC_CHUNKED_ARRAY_QUEUE;
+
+        static {
+            Object unsafe = null;
+            System.out.println(hasUnsafe());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS106)